content/organizations/managing-saml-single-sign-on-for-your-organization/configuring-saml-single-sign-on-and-scim-using-okta.md

```diff
@@ -31,8 +31,13 @@ After you enable SCIM, the following provisioning features are available for any
 
 Alternatively, you can configure SAML SSO for an enterprise using Okta. SCIM for enterprise accounts is only available with Enterprise Managed Users. For more information, see "[Configuring SAML single sign-on for your enterprise using Okta](/admin/identity-and-access-management/managing-iam-for-your-enterprise/configuring-saml-single-sign-on-for-your-enterprise-using-okta)" and "[Configuring SCIM provisioning for Enterprise Managed Users with Okta](/admin/identity-and-access-management/managing-iam-with-enterprise-managed-users/configuring-scim-provisioning-for-enterprise-managed-users-with-okta)."
 
-## Adding the {% data variables.product.prodname_ghe_cloud %} application in Okta
+## Configuring SAML for a {% data variables.product.prodname_ghe_cloud %} organization in Okta
```

Comment:
Based on your clarification in https://github.com/github/docs/pull/22476#issuecomment-1342959706, I agree that specifying SAML is a good idea. However, we need to keep heading titles as short as possible to increase scanability and reduce the cognitive load of reading the article. As I mentioned in https://github.com/github/docs/pull/22476#issuecomment-1343011987, it would be great if you could open an issue to internally discuss other ways how we clarify that this entire article relates to organizations only. 🙇🏻‍♀️ 

```suggestion
## Configuring SAML in Okta
```